app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/CheckAgencyController.scala

```diff
@@ -35,6 +35,15 @@ import uk.gov.hmrc.play.frontend.controller.FrontendController
 
 import scala.concurrent.Future
 
+object CheckAgencyController {
```

Comment:
not sure about this object, greatly confuses things and it's use is only for the knownfacts form. I recommend putting back into main controller and using `val controller = new CheckAgencyController(Dependency,Dependency)`to test the knowfacts form. Or at least change the object name?